src/cmd/intents/list/config.go

```diff
@@ -0,0 +1,5 @@
+package list
+
+const EnvironmentIDKey = "env-id"
+const IntentClientIDKey = "intent-client-id"
```

Comment:
why not client-service-id and server-service-id? (as opposed to having the `intent` prefix)

Reply:
Also please create a task to add the missing filters here once the intents API task is finished